validator/lib/validateTileset.js

```diff
@@ -35,8 +35,21 @@ function validateTopLevel(tileset) {
     if (!defined(tileset.geometricError)) {
         return 'Tileset must declare its geometricError as a top-level property.';
     }
+
+    if(!defined(tileset.asset)){
+        return 'Tileset must declare its asset as a top-level property.';
+    }
+
+    if(!defined(tileset.asset.version)){
+        return 'Tileset must declare its version property contained inside its top-level property asset.';
+    }
+
+    if(tileset.asset.version !== '1.0'){
+        return 'Tileset must declare its asset.version property as always set to 1.0.';
```

Comment:
Shorten to `Tileset version must be 1.0`. Also include in the message the version it saw.